packages/app/src/components/add-tag/AddTagStepTrack.vue

```diff
@@ -52,45 +79,63 @@ export default defineComponent({
   emits: ['update:nfc-tag'],
   setup(props, ctx) {
     const search = ref<string>('');
+    const isLoading = ref<boolean>(false);
     const tracks = ref<SpotifyTrack[]>([]);
     const tag = ref<NFCTag>(props.nfcTag);
     const selectedTrack = ref<SpotifyTrack>();
-    const info = ref<string>('');
-
-    watch(search, async (search) => {
-      if (search.length >= 3) {
-        const params = {
-          query: {
-            name: search,
-          },
-        };
-        tracks.value = (await feathers.service('spotify-tracks').find(params)) as SpotifyTrack[];
-        if (tracks.value.length === 0) {
-          info.value = `keine Ergebnisse für ${search}`;
-        } else {
-          info.value = 'Passende Songs zu deiner Suche';
-        }
-      } else if (search.length === 0) {
-        tracks.value = [];
-        info.value = 'Bitte gebe einen Songtitel ein';
+
+    const doSearch = debounce(async () => {
+      const params = {
+        query: {
+          name: search.value,
+        },
+      };
+      tracks.value = (await feathers.service('spotify-tracks').find(params)) as SpotifyTrack[];
+      isLoading.value = false;
+    }, 1000 * 0.5);
+
+    watch(search, async (_search) => {
+      if (_search.length >= 3) {
+        isLoading.value = true;
+        await doSearch();
       }
     });
 
     const changeTrack = (track: SpotifyTrack): void => {
       selectedTrack.value = track;
-      tag.value.spotifyTrackUri = track.uri;
-      tag.value.imageUrl = track.imageUri;
-      ctx.emit('update:nfc-tag', tag);
+
+      const tagCopy = tag.value;
+      tagCopy.spotifyTrackUri = track.uri;
+      tagCopy.imageUrl = track.imageUri;
+
+      ctx.emit('update:nfc-tag', tagCopy);
     };
 
     return {
       search,
+      isLoading,
       tracks,
       changeTrack,
-      info,
+      selectedTrack,
     };
   },
 });
 </script>
 
-<style></style>
+<style scoped>
```

Comment:
general question: 
Are we okay with writing small sections of css in file when using tailwind? Or would it be nicer if you create a new class in the tailwindconfig an use this instead.
personaly, i would like the second variation more.

Reply:
I would say if we just need something once we could use the style section

Reply:
This is a very special style I set here. We should not put that into tailwind.